pkgs/servers/monitoring/prometheus/node-exporter.nix

```diff
@@ -1,6 +1,6 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ stdenv, buildGo19Package, fetchFromGitHub }:
 
-buildGoPackage rec {
+buildGo19Package rec {
```

Comment:
Can you please add a comment with a reference to the upstream bug ?

Reply:
Done!